popup.js

```diff
@@ -0,0 +1,157 @@
+const submitBtn = document.getElementById('submit-btn')
+const addProjBtn = document.getElementById('addproject-btn')
+const urlBtn = document.getElementById('url-btn')
+const addProjKeyInput = document.getElementById('addProjectKey-input')
+const addProjKeyBtn = document.getElementById('addProjectKey-btn')
+const cancelBtn = document.getElementById('cancel-btn')
+const keysModal = document.getElementById('keys-modal')
+const urlModal = document.getElementById('url-modal')
+const keyList = document.getElementById('keyList')
+const modalBackground = document.getElementById('modal-background')
+const issueIdInput = document.getElementById('id-input')
+const urlComposition = document.getElementById('url-composition')
+const projectKeySelect = document.getElementById('project-select')
+const saveJiraUrlBtn = document.getElementById('save-url-btn')
+const deleteBtn = document.getElementById('delete-btn')
+const companyInput = document.getElementById('company-input')
+const jiraBaseUrl = document.getElementById('jira-url-input')
+//let jiraUrl = ""
+let storedKeys = ["1"]
+
```

Comment:
I added all eventListeners here as calling JS from within `popup.html` is not allowed in chrome extensions. It looks messy. Is this how it's done usually?